frontend/src/components/forms/CreateLog.tsx

```diff
@@ -156,8 +193,27 @@ const CreateLog = () => {
     setNotesError(inputValue === "");
   };
 
-  const [isCreateOpen, setCreateOpen] = React.useState(false);
+  // fetch resident + employee data for log creation
+  const getLogEntryOptions = async () => {
+    const residentsData = await ResidentAPIClient.getResidents()
+
+    if (residentsData) {
+      const residentLabels: NewSelectOptionType[] = JSON.parse(residentsData).map((r: any) => 
```

Comment:
This any should also be removed as a result of adding the types to the API clients